config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: kserve/kserve-controller:latest
+      - image: pvaneckw/kserve-controller:latest
```

Comment:
you can run `make deploy-dev` which should patch the image automatically for you